app/components/work_show_info_component.rb

```diff
@@ -128,6 +128,17 @@ def oral_history_number
     @oral_history_number ||= work.external_id.find { |id| id.category == "interview"}&.value
   end
 
+  def more_like_this_getter
+    @more_like_this_getter ||= MoreLikeThisGetter.new(work)
+  end
+
+  def more_like_this_works
+    more_like_this_getter.works
+  end
+
+  def more_like_this_json
+    more_like_this_getter.json
+  end
```

Comment:
I don't love that the component is triggering the call to Solr, so a call to Solr happens when you ask the Component to display. 

In an ideal world, the call to Solr would happen in the controller, and the data would be passed down to the component from the controller, somehow... but there may not be a non-terrible way to do this. It's worth spending some time at least thinking about it though. 

If we can't find a feasible way to do it, comments at teh top of this component should clearly advertise that it triggers a network call to solr for more like this results, via the MoreLikeThisGetter object.